src/main/kotlin/com/example/ajaxproject/controller/GroupChatController.kt

```diff
@@ -48,7 +48,7 @@ class GroupChatController @Autowired constructor(
     }
 
     @PostMapping("/leaveChat")
-    fun leaveChat(@RequestBody chatDto: ChatDTO): ResponseEntity<Boolean> {
+    fun leaveChat(@RequestBody chatDto: GroupChatDTO): ResponseEntity<Boolean> {
         return ResponseEntity.ok(groupChatService.leaveGroupChat(chatDto.chatId, chatDto.senderId))
```

Comment:
alternatively, you can use the body expression instead of "return"
   
    fun leaveChat(@RequestBody chatDto: GroupChatDTO): ResponseEntity<Boolean> = 
        ResponseEntity.ok(groupChatService.leaveGroupChat(chatDto.chatId, chatDto.senderId))

Reply:
@VolodymyrM27 please remind me about this thread on our next daily meeting

Reply:
discussed